tests/functional/disaster-recovery/regional-dr/test_disable_dr.py

```diff
@@ -0,0 +1,96 @@
+import logging
+import pytest
+from time import sleep
+
```

Comment:
empty line

Reply:
> empty line

empty line? To leave an empty line or remove line 4?